tutorials/issue1235.ipynb

```diff
@@ -0,0 +1,494 @@
+{
```

Comment:
<p>I like this figure and explanation. Please indicate what's a fracture and what is an interface, though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>